build-recipe-kiwi

```diff
@@ -507,6 +507,34 @@ is_legacy_kiwi() {
     return 0
 }
 
+get_kiwi_version() {
+    # lookup version number of installed kiwi version and
+    # turn it into a reliable number
+    local version
+    local major
+    local minor
+    local patch
+    if is_legacy_kiwi; then
```

Comment:
There is a is_legacy_kiwi method in build/build-recipe-kiwi, existing for some time and already in use. Thus I think it's not advisable to change the working part ;)